src/runtime_src/tools/xclbinutil/SectionSoftKernel.h

```diff
@@ -24,14 +24,15 @@
 class SectionSoftKernel : public Section {
  public:
   enum SubSection{
-    SS_UNKNOWN,
-    SS_OBJ,
-    SS_METADATA
+    UNKNOWN,
+    OBJ,
+    METADATA
   };
+ public:
+  static enum SubSection getSubSectionEnum(const std::string _sSubSectionName);
```

Comment:
```suggestion
  static enum SubSection getSubSectionEnum(const std::string& _sSubSectionName);
```

Reply:
Corrected this issue and similiar ones in the code base.